src/js/components/Avatar/Avatar.js

```diff
@@ -12,11 +12,13 @@ const Avatar = ({
   round = 'full',
   size = 'medium',
   src,
+  textSize,
```

Comment:
I don't think we need to extend the component API if we are providing an extensive theming option.